bluepyopt/deapext/algorithms.py

```diff
@@ -69,13 +68,22 @@ def _get_offspring(parents, toolbox, cxpb, mutpb):
     return deap.algorithms.varAnd(parents, toolbox, cxpb, mutpb)
 
 
+def _get_median(population, percentage=0.3):
+    '''return the median of the X% fittest individuals'''
+    _ = [ind.fitness.sum for ind in population]
+    _.sort()
+    return numpy.median(_[:int(percentage * len(_))])
+
+
 def eaAlphaMuPlusLambdaCheckpoint(
         population,
         toolbox,
         mu,
         cxpb,
         mutpb,
         ngen,
+        stagnation=0,
```

Comment:
Please put these at the end. If people use positional arguments i might break their scripts